qa/suites/orch/cephadm/workunits/task/test_iscsi_pids_limit.yaml

```
@@ -0,0 +1,20 @@
+roles:
+- - host.a
+  - osd.0
+  - osd.1
+  - osd.2
+  - mon.a
+  - mgr.a
+  - client.0
+tasks:
+- install:
+- cephadm:
+- cephadm.shell:
+    host.a:
+      - ceph osd pool create foo
+      - rbd pool init foo
+      - ceph orch apply iscsi foo u p
+- workunit:
+    clients:
+      client.0:
+        - cephadm/test_iscsi_pid_limit.sh
```

This must be a relative path to the workunit script:
```suggestion
        - cephadm/test_iscsi_pids_limit.sh
```